src/views/tickets/orders.php

```diff
@@ -31,7 +31,7 @@
 $post_type                  = get_post_type_object( $event->post_type );
 $user_id                    = get_current_user_id();
 $provider_id                = Tribe__Tickets__Tickets::get_event_ticket_provider( $event_id );
-$provider                   = call_user_func( array( $provider_id, 'get_instance' ) );
+$provider                   = call_user_func( [ $provider_id, 'get_instance' ] );
```

Comment:
Any change in a view requires a `@since` / `@version` update.

Reply:
OK, i'll back this out then. I'd prefer less noise in the template's changelog for just one array() to []